src/components/tracked-controls.js

```diff
@@ -84,7 +85,7 @@ module.exports.Component = registerComponent('tracked-controls', {
     var controllerQuaternion = this.controllerQuaternion;
     var controllerEuler = this.controllerEuler;
     var deltaControllerPosition = this.deltaControllerPosition;
-    var hand = (controller ? controller.hand : undefined) || DEFAULT_HANDEDNESS;
+    var hand = this.data.hand || (controller ? controller.hand : undefined) || DEFAULT_HANDEDNESS;
```

Comment:
Yes, the idea was that you could manually override any controller.hand defined property.  But I'm OK with removing that as per other comment about not giving the application a option to configure.